Formula/ctpv.rb

```diff
@@ -0,0 +1,23 @@
+class Ctpv < Formula
+  desc "Image previews for lf file manager"
+  homepage "https://github.com/NikitaIvanovV/ctpv"
+  url "https://github.com/NikitaIvanovV/ctpv/archive/refs/tags/v1.1.tar.gz"
+  sha256 "29e458fbc822e960f052b47a1550cb149c28768615cc2dddf21facc5c86f7463"
+  license "MIT"
+
+  depends_on "libmagic"
+  depends_on "openssl@3"
+
+  fails_with :clang do
+    build 1300
+    cause "Requires Clang 14 or later"
+  end
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/ctpv -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula